pkg/querier/distributor_queryable.go

```diff
@@ -129,15 +132,28 @@ func (q *distributorQuerier) streamingSelect(ctx context.Context, minT, maxT int
 		sets = append(sets, newTimeSeriesSeriesSet(results.Timeseries))
 	}
 
+	traceID, _ := tracing.ExtractTraceID(ctx)
```

Comment:
Could we move this inside the `if` two lines below? I don't think `traceID` is used anywhere else.